jax/_src/numpy/lax_numpy.py

```diff
@@ -6322,28 +6325,29 @@ def corrcoef(x, y=None, rowvar=True):
 
 @_wraps(np.quantile, skip_params=['out', 'overwrite_input'])
 @partial(jit, static_argnames=('axis', 'overwrite_input', 'interpolation',
-                               'keepdims'))
+                               'keepdims', 'method'))
 def quantile(a, q, axis: Optional[Union[int, Tuple[int, ...]]] = None, out=None,
-             overwrite_input=False, interpolation="linear", keepdims=False):
+             overwrite_input=False, method="linear", keepdims=False,
+             interpolation=None):
```

Comment:
Should we raise a deprecation warning if `interpolation` is not `None`?

Reply:
Yes, but I'm leaving that for a future PR: this is just to unbreak the CI.

Reply:
Changed my mind and did it.